client_apis/python/src/tests/integration/basetest.py

```diff
@@ -0,0 +1,35 @@
+#
+# CARBON BLACK API TESTS - watchlist
+# Copyright, Bit9, Inc 2015
+#
+
+""" These tests require CarbonBlack Enterprise Server to be installed.
+    You can run this script by passing the server URL and user API token
+    as parameters.
+"""
+
+import unittest
+import sys
+import os
+
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")))
+
+from cbapi.cbapi import CbApi
+
+class CbApiIntegrationTest(unittest.TestCase):
+    SERVER_URL = None
+    API_TOKEN = None
+
+    cb = None
+
+    @classmethod
+    def setUpClass(cls):
+        # instantiate a global CbApi object
+        # all unit tests will use this object
```

Comment:
docstrings are recommended instead of comments. This SO response clarified things for me http://stackoverflow.com/a/24385103/1212315, specifically reST (which is the defacto standard).